addons/mail/static/src/new/core/messaging.js

```diff
@@ -217,6 +225,22 @@ export class Messaging {
         });
     }
 
+    /**
+     * @param {Object} currentGuest
+     * @param {Object} currentPartner
+     * @param {integer} currentUserId
+     * @param {Object} partnerRoot
+     */
+    initUsers({ currentGuest, currentPartner, partnerRoot }) {
+        if (currentPartner) {
+            Partner.insert(this.state, currentPartner);
+        }
+        if (currentGuest) {
+            this.state.currentGuest = Guest.insert(this.state, currentGuest);
+        }
+        this.state.partnerRoot = Partner.insert(this.state, partnerRoot);
+    }
+
```

Comment:
Not worth the extraction when few LOCs, used only once, and lots of named args passed to function.
Would just dump these LOCs inside `initialize`